tests/gridworld_test.py

```diff
@@ -150,8 +154,8 @@ def test_equality(self):
     def test_dumploadloop(self):
         assert self.X == gw.GridWorld(self.X.dumps())
 
-    @unittest.skipIf(psutil.virtual_memory().total < 64e9,
-                     "not enough memory available")
+    @unittest.skipIf(dd_cudd is None,
```

Comment:
Excellent. This is a better solution because it allows the test to be performed on memory-constrained providers such as Travis CI.